docs/api/en/scenes/Fog.html

```diff
@@ -10,7 +10,7 @@
 	<body>
 		<h1>[name]</h1>
 
-		<p class="desc">This class contains the parameters that define linear fog, i.e., that grows linearly denser with the distance.</p>
+		<p class="desc">This class contains the parameters that define standard fog, i.e., that grows smoothly denser from the near to the far distance.</p>
```

Comment:
I am not sure about calling this "standard", but I really panicked when I saw that it wasn't linear at all. The smoothstep function is a cubic Hermite spline with clamped input.

Reply:
https://www.google.com/search?q=plot+t+*+t+*+%283.0+-+2.0+*+t%29+from+0+to+1

Reply:
Now described as "smooth ranged fog" with the latest commit. Any better name? Maybe this is good enough for now?

Reply:
Decided with https://github.com/mrdoob/three.js/issues/17420#issuecomment-534588800 (changes needed here)